maze-port.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/activity.py b/activity.py
+index 5eb236a..d254c77 100755
+--- a/activity.py
++++ b/activity.py
+@@ -11,9 +11,7 @@ from gi.repository import GLib
+ from gi.repository import Gdk
+ from gi.repository import Gtk
+ 
+-from sugar3.activity import activity
+ from sugar3.presence.presenceservice import PresenceService
+-from sugar3.activity.widgets import ActivityToolbarButton
+ from sugar3.activity.widgets import StopButton
+ from sugar3.graphics.toolbarbox import ToolbarBox
+ from sugar3.graphics.toolbutton import ToolButton
+@@ -26,19 +24,18 @@ from gettext import gettext as _
+ from textchannel import TextChannelWrapper
+ import game
+ 
++from sugarapp.widgets import SugarCompatibleActivity
++from sugarapp.widgets import ExtendedActivityToolbarButton
```

Comment:
Now that you updated to the latest Sugarapp version you don't need `ExtendedActivityToolbarButton` anymore.